scrapers/mt/actions.py

```diff
@@ -2,358 +2,131 @@
 
 
 # ----------------------------------------------------------------------------
-# Data for action categorization.
-
-_categories = {
+# Dictionary matching bill action phrases to classifications. Classifications can be found here:
+# https://github.com/openstates/openstates-core/blob/5b16776b1882da925e8e8d5c0a07160a7d649c69/openstates/data/common.py#L87
+# The "regex" included in some actions denotes that regex should be used to identify those phrases
+_actions = {
     # Bill is introduced or prefiled
-    "introduction": {"rgxs": ["^Introduced$"], "funcs": {}},
-    # Bill has passed a chamber
-    "passage": {
-        "rgxs": [
-            "3rd Reading Passed",
-            "^Resolution Adopted",
-            "3rd Reading Concurred",
-            "3rd Reading Passed as Amended by Senate",
-            "3rd Reading Passed as Amended by House",
-        ]
-    },
+    "^introduced$": ["introduction", "regex"],
```

Comment:
Is it worth making this format something like:
```
"^introduced$": {"type": "regex", "mappings": ["introduction"]},
"3rd reading passed": {"type": "compare", "mappings": ["passage", "reading-3"]},
...
```
This lets us avoid the whole `_actions[key][-1]` to check for `regex`.

So then processing looks something like:

```
def categorize_actions(act_desc):
    atype = []
    for action_key in _actions.keys():
        # If the action requires regex to be correctly classified
        if _actions[action_key]["type"] == "regex":
            if re.search(action_key, act_desc.lower()):
                # Add all action classifications in the mappings list
                atype.extend(a for a in _actions[action_key]["mappings"])
        else:
            if action_key.lower() in act_desc.lower():
                atype.extend(a for a in _actions[action_key]["mappings"])
```